reana_db/models.py

```diff
@@ -663,10 +663,29 @@ class Job(Base, Timestamp):
     env_vars = Column(JSONType)
     deleted = Column(Boolean)
     logs = Column(String, nullable=True)
+    started_at = Column(DateTime)
+    finished_at = Column(DateTime)
```

Comment:
Do we have Alembic upgrade recipe for this change? Or shall we compile them afterwards before releasing?

Reply:
Definitely needed.